web/handlers/admin/handler.go

```diff
@@ -63,11 +63,30 @@ func Handler(
 	// TODO: configure 404 handler
 
 	mux.HandleFunc("/dashboard", r.HTML("admin/dashboard.tmpl", func(rw http.ResponseWriter, req *http.Request) (interface{}, error) {
-		lst := roomState.List()
+		onlineRefs := roomState.List()
+		onlineCount := len(onlineRefs)
+		members, err := dbs.Members.List(req.Context())
+		memberCount := 0
+		if err == nil {
+			memberCount = len(members)
+		}
```

Comment:
All this stuff should be converted to `.Count()` function on the interfaces. I tried to do that but got a bit confused with the generated Go code and how to handle all that.

Reply:
Is there any utility in returning the error? I'm thinking that this method could just return zero in those cases.

Reply:
> Is there any utility in returning the error?

It borders on philosophical but I prefer to see those and not render the page instead of assuming no-one is connected, etc.

One possible error if a transaction bug sneaked is `sqlite: database is locked`.

Reply:
My general stance is, if errors aren't handled they need to be logged but if no-one is reading the log output that's more disservice of the software.